cypress/integration/admin.js

```diff
@@ -26,4 +26,22 @@ describe('Sourcebooks admin', function() {
         check.searchTaskName('[type="text"]', task[0]);
         check.visibilityCheck('[role="row"]');
     })
+
+    it('Admin creates a client', function() {
+        var task = [Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15), Math.random().toString(36).substring(2, 15)]
+        task[3] += "@gmail.com";
+        console.log(task[3]);
+
+        check.click('[href="/clients"]');
+        check.click('.btn');
+        check.inputText('[id="clientDetailsForm.organization"]' ,task[0]);
```

Comment:
locators are defined in a test case, this is not good practice according to page object model